alma/alma-installments-magento1
# feature : Add hmac validation to IPN controller

https://linear.app/almapay/issue/ECOM-1823/[-magento1]-add-hmac-verification-on-ipn

Add verification signature in IPN action. 

How to test :
Must be tested in QA env ! 
Create a product and set Alma **test & live** key
Create a paiement and check ipn logs in Alma dashbord

Use this payment id to check ipn on route : 
{baseURL}alma/payment/ipn?pid={paymentId}
without signature header and with a bad signature.

No unit test on magento 1 for the moment.
